getnacelle/nacelle-react
# fix: remove lockfiles to avoid symlinks in VC

**Story:** [LS-1393](https://nacelle.atlassian.net/browse/LS-1393)

### Type of Change


### What is being changed and why?



In #66, `package-lock.json` files were added to source control. This created problems, because symlinks created by `lerna bootstrap` led to problems during deployment and when scaffolding a new project from `examples/*`.

This PR keeps `packages/` and `examples/`' lock files out of version control. The root-level `package-lock.json` is still tracked, as described here: https://www.linkedin.com/pulse/things-i-wish-had-known-when-started-javascript-monorepo-gore.

### How to Test



Deploy previews for `examples/nextjs` should deploy without issues.

_Update: 👍 (https://nacelle-nextjs-example-git-ls-1393-remove-lockfiles-nacelle.vercel.app)_

To test locally:
```
npx degit https://github.com/getnacelle/nacelle-react/examples/nextjs\#LS-1393-remove-lockfiles nextjs-demo
cd nextjs-demo
npm i
```
Next, add a `.env` based on the variables outlined in `.env.example` (I can DM you one in Slack).

Finally, `npm run dev`, and navigate to [`localhost:4000`](http://localhost:4000), where the Next.js site should be available .
